packages/gasket-plugin-happyfeet/README.md

```diff
@@ -0,0 +1,56 @@
+# @gasket/plugin-happy-feet
+
+A plugin that creates a [happy-feet](https://github.com/asilvas/happy-feet#usage) instance and adds pod memory healthchecks based on config that you provide.
```

Comment:
Should be generalized since it does more than monitor memory, and it works for more than just Kubernetes.

```suggestion
Integrates with [happy-feet](https://github.com/asilvas/happy-feet#usage) to add application instance termination when health metrics such as memory usage enter a bad state.
```